R/crossvalidation.R

```diff
@@ -185,22 +197,27 @@ crossvalidation <- function(formula,
   test <- list()
   for (i in 1:k) {
     test[[i]] <- data[folds == i  ,]
-  }
-  
+    }
   }
   
   #folds as list - caret style
-  if(is.list(folds)){
+  if (is.list(folds)) {
+    
+    k <- length(folds)
+    mean.method <- "equal"
+    message("\nmean.method set to 'equal' as folds is a list \n")
```

Comment:
I think this apply if the folds are more or less of equal size (random k-fold) but it is not always the case when is blocked CV. Perhaps we should leave the user to decide with a warning message in the case of random k-folds (not blocked).